index.ts

```diff
@@ -26,8 +26,8 @@ import storedCredentials from './credentials.json';
   })() as Required<ICredentials>;
 
   const certification = await getCertification(credentials);
-  const { data: html } = await getCheckResponse({ certification, ...credentials });
-  if (html.includes(successText)) {
+  const { data } = await getCheckResponse({ certification, ...credentials });
+  if(data.resultSVO.rtnRsltCode == successText) {
```

Comment:
```suggestion
  const { data: { resultSVO: { rtnRsltCode: resultCode }  } = await getCheckResponse({ certification, ...credentials });
  if(resultCode == successText) {
```

구조 분해 할당 문법을 사용하면 더 가독성 있는 코드가 나옵니다!

Reply:
수정했습니다 :)